api/lib/infrastructure/caches/DistributedCache.js

```diff
@@ -15,6 +16,7 @@ class DistributedCache extends Cache {
       this._redisClientSubscriber.subscribe(this._channel);
     });
     this._redisClientSubscriber.on('message', () => {
+      logger.info('Flushing the local cache');
```

Comment:
```suggestion
      logger.info({event: 'cache-flush' },'Flushing the local cache');
```
Si on veut se simplifier les recherches après on pourrait mettre un event ici

Reply:
Je fais la même pour l'autre event en unifiant le type: `cache-event` ou un truc du genre ?